### DISCUSSION TITLE
SDM72D-M Smartmeter reading over Modbus

### DISCUSSION THREAD

**Discussion by MarioST2023** [benzino77/tasmocompiler#359] [General]
Hello,

I compile tasmota over Tasmocompiler. i choose " Energiemeter", "ModbusBridg" and " Modbus TTL" but the only  Energiemeter at Modulparameter are SDMX20 and SDM630.

can someone help me?

**Comment by benzino77** [comment]
Click on the anchor (selected with red circle). This will bring you to the description of what sensors are included in each feature on the list.
![features_desc](https://github.com/benzino77/tasmocompiler/assets/15658235/60bdf1df-f23b-4c0f-8c40-7a05a468e838)

If there is no sensor you need on the list you have couple of options:
1. Make a PR, where you add additional sensors to the TasmoCompiler so others can also use it in the future
2. Use "Custom Parameters" field in step 4 and add all `#define` you need to include sensors/features you want, which are not currently included in TasmoCompiler

**Reply by MarioST2023** [reply]
i have just write in this field " Costum Parameters" #define USE_SDM72 ?
